KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -0,0 +1,81 @@
+# [kubevirtci](README.md): Testing kubevirtci with locally provisioned cluster
+
+After making changes to a kubevirtci provider, you should test it locally before publishing it.
+
+Let's go on the required steps, starting by provisioning, all the way to `make cluster-up`.
+`$KUBEVIRTCI_DIR` is assumed to be your kubevirtci path.
+
+Steps:
+
+### kubevirtci: provision cluster locally
+
+```bash
+cd $KUBEVIRTCI_DIR
+cd cluster-provision/k8s/1.19
+# Build a provider. This includes starting it with cluster-up for verification and shutting it down for cleanup.
+../provision.sh      
```

Comment:
```suggestion
# Build a provider. This includes starting it with cluster-up for verification and shutting it down for cleanup.
(cd cluster-provision/k8s/1.19; ../provision.sh)
```
I prefer this, because nothing else needs to be run from `cluster-provision/k8s/1.19` dir

Reply:
Done